src/main/java/it/gov/pagopa/arc/controller/TransactionsControllerImpl.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.arc.controller;
+
+
+import it.gov.pagopa.arc.dto.TransactionDTO;
+import it.gov.pagopa.arc.service.TransactionsService;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class TransactionsControllerImpl implements TransactionsController {
+    private final TransactionsService transactionsService;
+
+    public TransactionsControllerImpl(TransactionsService transactionsService) {
+        this.transactionsService = transactionsService;
+    }
+
+    @Override
+    public List<TransactionDTO> getTransactionsList(String fiscalCode, String continuationToken, int size) {
```

Comment:
why are continuationToken and size not used?

Reply:
I inserted this for future implementation with the real service, right now this data is not used because I use a dummy json response

Reply:
let's try to keep the service interface stable. The difference between this service implementation and the future one should regard only the service implementation itself. The controller implementation must not change.

Reply:
so I should keep these parameters since they will be used in the real implementation, correct ?